### ISSUE TITLE
Yogg Shadow Beacon auto-marking rolls over

### ISSUE THREAD

**Issue by Heaviside2** [DeadlyBossMods/DBM-WotLK#28]
Shadow Beacon marking should always start with Skull on each wave of beacons.
Instead, the self.vb.beaconIcon value is left alone after each wave, so subsequent waves use dissimilar marks.
The result is that taunters currently need to be assigned like this, with diff taunt marks assigned for each wave for your first 3 taunters:
Taunter 1: Wave 1 - Skull, Wave 2 - Moon
Taunter 2: Wave 1 - Cross, Wave 2 - Triangle
Taunter 3: Wave 1 - Square, Wave 2 - Diamond
Taunter 4: Wave 2 - Circle
Taunter 5: Wave 2 - Star

It would be better if DBM would always start marking adds from the top with Skull on each wave, so the assignments can be like this:
Taunter 1: Wave 1+2 - Skull
Taunter 2: Wave 1+2 - Cross
Taunter 3: Wave 1+2 - Square
Taunter 4: Wave 2 - Moon
Taunter 5: Wave 2 - Triangle

DBM 3.4.28

I think it would be sufficient to add this line right after #263:
			self.vb.beaconIcon = self.vb.beaconIcon + 1
Or possibly also these additional lines out of paranoia:
			if self.vb.beaconIcon > 8 then
				self.vb.beaconIcon = 8
			end

**Comment by MysticalOS** [comment]
Yeah I wondered about changing this, i was mostly just making sure the fix works first. make sure everything is at least marked, and marks clear when beacon ends. Can you confirm both of those issues fixed?

**Comment by Heaviside2** [comment]
Since your fix for https://github.com/DeadlyBossMods/DBM-WoTLKC/issues/11 was so recent, I can't confirm that yet.
Also, I think everyone has been using "my" WA (or one like it) to do the marking so far.

Also, perhaps just setting self.vb.beaconIcon = 8 after the spell auras get removed is safer than re-incrementing it.  But I'm sure you will know better than I.

**Comment by MysticalOS** [comment]
Yeah I"m gonna add a reset for tuesday.

**Comment by MysticalOS** [comment]
In fact, it's already done in alpha ;) So will go ahead and close this
